frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -236,15 +237,26 @@ export const supportLogic = kea<supportLogicType>([
                 .then((res) => res.json())
                 .then((res) => {
                     const zendesk_ticket_id = res.request.id
+                    const zendesk_ticket_link = `https://posthoghelp.zendesk.com/agent/tickets/${zendesk_ticket_id}`
                     const properties = {
                         zendesk_ticket_uuid,
                         kind,
                         target_area,
                         message,
                         zendesk_ticket_id,
-                        zendesk_ticket_link: `https://posthoghelp.zendesk.com/agent/tickets/${zendesk_ticket_id}`,
+                        zendesk_ticket_link,
                     }
                     posthog.capture('support_ticket', properties)
+                    Sentry.captureMessage('User submitted zendesk ticket', {
+                        tags: {
+                            zendesk_ticket_uuid,
+                            zendesk_ticket_link,
+                            support_request_kind: kind,
+                            support_request_area: target_area,
+                        },
+                        extra: properties,
+                        level: 'error',
```

Comment:
I don't think this is an error, tickets can also be support requests or random feedback

Reply:
Changed